index.js

```diff
@@ -0,0 +1,140 @@
+                    <div>
+                        <h4 class="speaker-tag">${teacher.name}</h4>
+                        <p class="speaker-title">
+                            ${teacher.position}
+                        </p>
+                    </div>
+                    <br>
+                    <div class="speaker-bio">
+                        <p class="name">${teacher.bio}</p>
+                    </div>
+                </div>
+            </div>
+            `,
+        )
+        .join('');
+      expand += 1;
+    } else {
+      const bot = document.querySelectorAll('.automated');
+      for (let i = 0; i < bot.length; i += 1) {
+        const item = bot[i];
+        item.parentNode.removeChild(item);
+      }
+      expand = 0;
+    }
+  });
+} else {
+  const tutor = document.querySelector('.speaker-container');
+  tutor.innerHTML += speakers.map((teacher) => `
+  <div class="speaker-info automated">
+                <div class="speaker-img">
+                    <img class="thumbnail1_1" src='${teacher.image}' alt="image of speaker">
+                </div>
+                <div class="bio-container">
+                    <div>
+                        <h4 class="speaker-tag">${teacher.name}</h4>
+                        <p class="speaker-title">
+                            ${teacher.position}
+                        </p>
+                    </div>
+                    <br>
+                    <div class="speaker-bio">
+                        <p class="name">${teacher.bio}</p>
+                    </div>
+                </div>
+            </div>
+            `).join('');
+}
+
+// dynamic featured speakers section
+// const tutor = document.querySelector('.speaker-container');
+// const moreTutors = document.querySelector('.expand-sp-list');
+
+// let expand = 0;
+// moreTutors.addEventListener('click', () => {
+//   if (expand === 0) {
+//     tutor.innerHTML += speakers.map((teacher) => `
+// <div class="speaker-info automated">
+//                 <div class="speaker-img">
+//                     <img class="thumbnail1_1" src='${teacher.image}' alt="image of speaker">
+//                 </div>
+//                 <div class="bio-container">
+//                     <div>
+//                         <h4 class="speaker-tag">${teacher.name}</h4>
+//                         <p class="speaker-title">
+//                             ${teacher.position}
+//                         </p>
+//                     </div>
+//                     <br>
+//                     <div class="speaker-bio">
+//                         <p class="name">${teacher.bio}</p>
+//                     </div>
+//                 </div>
+//             </div>
+//             `).join('');
+//     expand += 1;
+//   } else {
+//     const bot = document.querySelectorAll('.automated');
+//     for (let i = 0; i < bot.length; i += 1) {
+//       const item = bot[i];
+//       item.parentNode.removeChild(item);
+//     }
+//     expand = 0;
+//   }
+// });
```

Comment:
Please make sure to remove all the commented code from your project 😉. This will make your project more clear and easier to maintain.